.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,29 @@
+name: Github Pages
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
what's this?

Reply:
[concurrency](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#concurrency)

As far as I understand it just avoids the same workflow to be run at the same time, e.g. workflow triggered by a merge into `main` and someone manually requesting the CD workflow.

We can remove it though, if you say this is overkill/unnecessary here.

Reply:
I removed it now, you can decide whether you seem it fits or not. Since we do not trigger the workflow with pull requests it might really be superfluous.

Reply:
No it's a good idea to use it if it avoids duplicate runs. We have a limited number of free CI minutes per month ;)

Reply:
Inspired me to create https://github.com/perun-network/erdstall-ts-sdk/issues/123

Reply:
However, for `gh-pages` it is probably superfluous because, as you said, it would only be triggered manually or by a merge into main, both very conscious actions that shouldn't happen accidentally at the same time.